### DISCUSSION TITLE
no data showing in alerts/hunt

### DISCUSSION THREAD

**Discussion by wtl06639** [Security-Onion-Solutions/securityonion#2938] [Unsupported Versions]
I just set up a manager and a sensor. I enabled Suricata on the sensor but not zeek.  Everything looks to be ok and the sensor and manager are communicating but i dont see any data when i go to the manager webpage under the alerts section. 

I see traffic hitting the sensor span port. And checking /opt/so/log/suricata/stats.log looks like it is logging traffic. 

I see the sensor on the grid port of the manager web page and it is showing online. However no alerts are showing up.

**Comment by TOoSmOotH** [comment]
Did you set the manager up as manager+search? Otherwise you need to add a search node.

**Comment by meredib** [comment]
I'm actually going through a similar issue.  I have one master node and one heavy node.  If I navigate to on the Heavy Node to /opt/so/log/suricata/ and cat stats.log.  I see a number of items, specifically I notice <detect.alert | Total  | 912487>, which makes me think that the heavy node is working as expected. 

However, when I look at the heavynode dashboard of Grafana from the Master Node and look specifically at the Logstash Traffic of the Heavy Node sensor I see the following graph.

![Screen Shot 2021-02-17 at 9 09 26 AM](https://user-images.githubusercontent.com/6569426/108215921-db1cc800-70ff-11eb-8b8d-1e63b0ec3c62.png)

However, when I look at Kibana or Security Onion Alerts there doesn't appear to be anything there.

![Screen Shot 2021-02-17 at 9 11 13 AM](https://user-images.githubusercontent.com/6569426/108216157-2636db00-7100-11eb-99b7-eabbbf42f26a.png)

This is a brand new build similar to what the person who created this discussion was doing.

**Reply by LS3-Allen** [reply]
I have a similar issue.  On 7/9 I upgraded from 2.3.52 to 2.3.60 and I see no data in Hunt or Kibana since then.
